### DISCUSSION TITLE
Yorum

### DISCUSSION THREAD

**Discussion by tubbaasarikaya** [19/classroom#64] [C]
Hocam iyi akşamlar ödevle çok uğraştım ama bir yerlerde takılı kalıyorum sürekli. Yarın son gün . Yarın geceye kadar ugrasicam .

**Comment by roktas** [comment]
Önemli olan uğraşman.

**Comment by roktas** [comment]
Ama böyle de yardım istemen yararlı değil.  Kim nereye bakacak?  Nerede takıldın vs.  Ödev deponda "Feedback" isimli bir PR açık.  Orada yazsaydın hem yeri belli olurdu hem de belki makul bir yardım alabilirdin.

**Reply by tubbaasarikaya** [reply]
Haklısınız hocam . Fakat şöyle . C derslerine yeni yeni başladık yani yeni yeni uygulamalar yapmaya çalışıyoruz. İlk odeve göre bana biraz fazla kompleks yapıda geldi. Bir noktaya kadar geliyorum bazı şeyleri toparliyamiyorum. Bu benimle alakali bir sorun olduğu için size kodumla ilgili hataları paylaşmadım. İlk attığım yorumumun da sebebi şu . Ödev için gerçekten fazlasıyla uğraştım ama yarına yetistirebilcegimi düşünmüyorum emek verdiğimi bilin istedim. Teşekkür ederim.

**Comment by roktas** [comment]
Emek verdiğini göstermen için daha fazlası gerekmiyor mu?  Bak basit bir şey söyledim PR'da, hiç kod göndermemişsin diyorum.
